src/cascadia/QueryExtension/ExtensionPalette.cpp

```diff
@@ -373,6 +373,26 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
         _queryBox().Focus(FocusState::Programmatic);
     }
 
+    void ExtensionPalette::_exportMessagesToFile(const Windows::Foundation::IInspectable& /*sender*/,
+                                                 const Windows::UI::Xaml::RoutedEventArgs& /*args*/)
+    {
+        std::wstring concatenatedMessages{};
+        for (const auto groupedMessage : _messages)
+        {
+            groupedMessage.IsQuery() ? concatenatedMessages += RS_(L"UserString") : concatenatedMessages += RS_(L"AssistantString");
```

Comment:
what? this is absolutely not the way to write an `if`!

Reply:
restructure as either `concatenatedMessages += foo ? bar : baz;` or `if (foo) { ... }`

Reply:
FWIW you could also swap out the `const auto` with an `const auto&`.